mapbox_gl_web/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: mapbox_gl_web
 description: Web platform implementation of mapbox_gl
 version: 0.15.0
-homepage: https://github.com/tobrun/flutter-mapbox-gl
+homepage: https://github.com/Sipioteo/maps_3_0_0
```

Comment:
I guess this doesn't have to be in the changeset

Reply:
I thought i have removed it. Wait.